### PR TITLE
Update ru.json

### PR DESCRIPTION
**Pull request by hide-me**
### What does it do?

Added new lines from en.json

### Why is it needed?

To complete full translation of new lines.

### How to test it?

No need to test.

### CODE REVIEW

File: packages/core/admin/admin/src/translations/ru.json [hunk 1]
```diff
@@ -1,900 +1,939 @@
...
+	"Settings.permissions.auditLogs.payload": "Полезная нагрузка",
+	"Settings.permissions.auditLogs.permission.create": "Создал разрешение",
+	"Settings.permissions.auditLogs.permission.delete": "Удалил разрешение",
+	"Settings.permissions.auditLogs.permission.update": "Обновил разрешение",
+	"Settings.permissions.auditLogs.role.create": "Создал роль",
+	"Settings.permissions.auditLogs.role.delete": "Удалил роль",
+	"Settings.permissions.auditLogs.role.update": "Обновил роль",
+	"Settings.permissions.auditLogs.user": "Пользователь",
+	"Settings.permissions.auditLogs.user.create": "Создал пользователя",
+	"Settings.permissions.auditLogs.user.delete": "Удалил пользователя",
```
[the remark this answers is not in the document]
Comment (hide-me) [reply packages/core/admin/admin/src/translations/ru.json:177]: Thank you, I'll do it in the next pull request!

File: packages/core/admin/admin/src/translations/ru.json [hunk 2]
```diff
@@ -1,900 +1,939 @@
...
+	"Settings.apiTokens.description": "Список сгенерированных токенов для использования API",
+	"Settings.apiTokens.emptyStateLayout": "У вас ещё нет контента...",
+	"Settings.apiTokens.lastHour": "последний час",
+	"Settings.apiTokens.regenerate": "Перегенерировать",
+	"Settings.apiTokens.title": "API-токены",
+	"Settings.application.customization": "Персонализация",
+	"Settings.application.customization.auth-logo.carousel-hint": "Замените логотип на странице авторизации",
+	"Settings.application.customization.carousel-hint": "Изменить логотип панели администратора (максимальный размер: {dimension}x{dimension}, максимальный размер файла: {size}KB)",
+	"Settings.application.customization.carousel-slide.label": "Слайд с логотипом",
+	"Settings.application.customization.carousel.auth-logo.title": "Логотип на странице авторизации",
```
[the remark this answers is not in the document]
Comment (hide-me) [reply packages/core/admin/admin/src/translations/ru.json:105]: Why my translation is not accurate?

[the remark this answers is not in the document]
Comment (hide-me) [reply packages/core/admin/admin/src/translations/ru.json:105]: Anyway, let's make your variant

**Comment by hide-me** [comment]
All suggestions was good and accurate, thank you for it. Please accept this pull request.

**Review by madhurisandbhor — COMMENTED** [review]
LGTM, please resolve the conflicts

**Comment by egormkn** [comment]
I think it would be better to use indentation with two spaces instead of tabs, as in other translation files

**Comment by hide-me** [comment]
> it would be better to use indentation with two spaces instead of tabs
Yeah, but few translators uses tabs, not only me. It's more comfortable and I immediatly see if I miss tab.

**Comment by hide-me** [comment]
Merge it, please 🥺

**Comment by madhurisandbhor** [comment]
> Merge it, please 🥺

@hide-me Please resolve conflicts

**Comment by hide-me** [comment]
No, later, I'll make new pull reqest. Maybe

**Pull request closed without merging**